tensorboard/webapp/BUILD

```diff
@@ -55,10 +55,10 @@ ng_module(
 # compatibility, Vulcanization, and configuration (e.g. prod vs. dev).
```

Comment:
Comment here is stale now that it's not switching prod vs dev.

Is it actually fine to be using prod mode all the time, or is this just temporary until we can get it working again?  If the latter, maybe we should include a TODO?